### ISSUE TITLE
Panel - pool - number of workers

### ISSUE THREAD

**Issue by Photoshop-cs** [UnamSanctam/SilentCryptoMiner#1997]
Hello Unam.  Maybe this question has already been asked.  But I'll ask again.  In the panel and in the pool, the difference between online/total employees is 2 times.  That is, the pool shows 2 times fewer online workers.  All settings are correct.  Where do the rest of the workers go?

**Comment by Photoshop-cs** [comment]
Yes.  I have one domain for the GPU and another for the CPU.  But still, my pool shows fewer workers online than the panel.

**Comment by UnamSanctam** [comment]
For 2miners.com there shouldn't be a limit, for miningocean.org I don't know, they usually don't say. Most pools don't have a maximum though they can of course get suspicious if you have a lot of workers.
